tests/format/markdown/toml/format.test.js

```diff
@@ -1 +1,10 @@
-runFormatTest(import.meta, ["markdown"], { proseWrap: "always" });
```

Comment:
Let's also not name the directory as `toml`, maybe also `with-plugins`, since we may support more languages in future.